### ISSUE TITLE
JOSS Review - 11 -  numeric effect size measures rather than labels based on arbitrary thresholds

### ISSUE THREAD

**Issue by DominiqueMakowski** [neuropsychology/psycho.R#31]
Point 11 from [JOSS Review](https://github.com/openjournals/joss-reviews/issues/470#issuecomment-356727181):

--------------
 
The analyze() function is nice, but I don't really like the string labeling of effect sizes as "Very small", as I am always in favor of numeric effect size measures rather than labels based on arbitrary thresholds. Can't the numeric effect size be printed instead?

**Comment by DominiqueMakowski** [comment]
It is probably a reference to `analyze.merModLmerTest` and `analyze.stanreg`.

The former returns indeed a "labelled" effect size in `print`
```r
The overall model predicting ... successfully converged and explained 85.60% of the variance of the endogen (the conditional R2). The variance explained by the fixed effects was of 9.06% (the marginal R2) and the one explained by the random effects of 76.54%.
The effect of (Intercept) was [NOT] significant (beta = 3.41, SE = 0.67, t(3.41) = 5.10, p < .05*) and can be considered as very small (std. beta = 0, std. SE = 0).
The effect of Sepal.Width was [NOT] significant (beta = 0.80, SE = 0.11, t(146.66) = 7.51, p < .001***) and can be considered as small (std. beta = 0.42, std. SE = 0.056).
```
And `summary`:
```r
                 Coef        SE        t         df  Coef.std     SE.std            p Effect_Size
(Intercept) 3.4061671 0.6683079 5.096703   3.405329 0.0000000 0.00000000 1.065282e-02  Very Small
Sepal.Width 0.7971542 0.1062064 7.505711 146.664966 0.4195953 0.05590347 5.453415e-12       Small
```
However, this effect size is based on the value of the standardized coefficient, which value is also returned. One can always ignore the arbitrary labels and use the values instead.

-------
As for Bayesian models and the effect sizes present in `print`, e.g.:

 ```
Based on Cohen (1988) recommandations, there is a probability of 0% that this effect size is very large, 46.70% that this effect size is large, 53.02% that this effect size is medium, 0.27% that this effect size is small, 0% that this effect is very small and 0% that it has an opposite direction (between 0 and 0.45).
```

They are directly based on the coefficient distribution (the posterior), hence the need for the data to be standardized beforehand.

Again, one can always use the values instead of the labels.


------

However, I agree that it would be nice indeed to allow to user to pass its own labels-reference and customize the categorization of effect sizes.

**Comment by DominiqueMakowski** [comment]
First iteration done in #63
